binding/vue/README.md

```diff
@@ -12,7 +12,9 @@ applications.
 
 ## Compatibility
```

Comment:
# porcupine-web-vue

Renderless Vue component for Porcupine for Web. -> is this still a correct description for this binding?